### DISCUSSION TITLE
Feature Request: allow to add a comment on a specific word(s)/letter(s)

### DISCUSSION THREAD

**Discussion by YairoR** [community/community#18424] [Pull Requests]
Currently you can only add a comment in a PR against a specific line. My suggestion is to allow to add a comment against a specific letter(s)/word(s).

By having this ability, the reviewer will be able to point to a specific issue within the line; For example, if there is a typo in a comment, I'll be able to mark the specific mistake.

An example for implementation:
![image](https://user-images.githubusercontent.com/7312492/173231769-f27b1df6-daad-495c-b32f-30183ced54ff.png)

**Comment by kreintjes** [comment]
Can't you use code suggestions (` ```suggestion`) to achieve this? These currently are only available when viewing all Files changed, but there are feature requests to add them for single commits as well: https://github.com/github-community/community/discussions/7128
